components/serviceMap/nodeMenu.tsx

```diff
@@ -1,33 +1,30 @@
-import { PipelineInfo } from "https://deno.land/x/snitch_protos@v0.0.56/protos/info.ts";
 import { Edit, Info, Pause, Silence } from "../icons/crud.tsx";
 import IconDots from "tabler-icons/tsx/dots.tsx";
-import { useState } from "https://esm.sh/stable/preact@10.15.1/denonext/hooks.js";
-
-export const NodeMenu = ({ data }: { data: PipelineInfo }) => {
-  const [isOpen, setIsOpen] = useState();
-
-  const handleClick = () => {
-    setIsOpen(!isOpen);
-  };
+import { useState } from "preact/hooks";
```

Comment:
I tried to clean up imports whenever possible...using our aliases so we can upgrade centrally in the future.